health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateCampaignComponents/QRButton.js

```
@@ -0,0 +1,68 @@
+          equalWidthButtons={"false"}
+          footerChildren={[
+            <Button
+              // className={"campaign-type-alert-button"}
+              type={"button"}
+              size={"large"}
+              variation={"primary"}
+              label={t("ES_CAMPAIGN_CLOSE")}
+              onClick={() => {
+                setShowQRPopUp(false);
+              }}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)